core/css.js

```diff
@@ -150,8 +150,7 @@ Blockly.Css.CONTENT = [
     'box-shadow: 4px 4px 20px 1px rgba(0,0,0,.15);',
```

Comment:
This is independent of any other css, in the spirit to keep module and css together, maybe this could be moved to ``tooltip.js``.

Reply:
True.  Though it is only one CSS rule, and tooltips aren't an optional module.